content/verification/veteran-confirmation/release-notes/2024-05-14.md

```diff
@@ -0,0 +1,24 @@
+Veteran Confirmation
+/status
```

Comment:
Is status supposed to be listed here?

Reply:
What we're trying to convey here is that these changes are for post requests of /status, but that's also the only thing you can do with Veteran confirmation. 
![image](https://github.com/department-of-veterans-affairs/developer-portal-content/assets/153761193/2dcc2925-00bf-4711-8c40-feecef272e09)
Maybe there is a better way that we could word or even remove /status and the Post requests - so that it just starts with:

Special characters and spaces are now accepted.... and so on